### ISSUE TITLE
'Other bookmarks' translation missing

### ISSUE THREAD

**Issue by Johann999** [brave/brave-browser#9218]
Since version 1.7.92 the folder 'Other bookmarks' is again in the right place in the bookmarks bar. Many thanks.
But I miss the translation into the language in which Brave is carried out.

**Comment by rebron** [comment]
@darkdh Looks like `Other bookmarks` isn't translated. Can we hook up to the already translated chromium strings?

**Comment by btlechowski** [comment]
This issue is still reproducible on 1.9.x, 1.10.x and 1.11.x

![image](https://user-images.githubusercontent.com/34715963/83175092-abfe9a80-a11b-11ea-8b57-8bc43eea3483.png)

![image](https://user-images.githubusercontent.com/34715963/83175244-e5370a80-a11b-11ea-8b9b-929b57f2a466.png)

Brave | 1.11.31 Chromium: 83.0.4103.61 (Official Build) nightly (64-bit)
-- | --
Revision | 94f915a8d7c408b09cc7352161ad592299f384d2-refs/branch-heads/4103@{#561}
OS | Windows 7 Service Pack 1 (Build 7601.24544)

**Comment by Johann999** [comment]
When will this bug be finally corrected?

**Comment by darkdh** [comment]
I've never changed the translation of "Other Bookmarks" or "Mobile Bookmarks" which are both missing.

Checking this with @mkarolin,
Somehow we only have these translation left and even `en-GB.xtb` version is missing
```
components/strings/components_strings_es.xtb:232:<translation id="2354001756790975382">Otros marcadores</translation>
components/strings/components_strings_nl.xtb:231:<translation id="2354001756790975382">Andere bladwijzers</translation>
components/strings/components_strings_ru.xtb:239:<translation id="2354001756790975382">Другие закладки</translation>
components/strings/components_strings_sq.xtb:231:<translation id="2354001756790975382">Faqeruajtës të tjerë</translation>
components/strings/components_strings_zh-CN.xtb:223:<translation id="2354001756790975382">其他书签</translation>
```
And we can't see original chromium files on Transifex right now because we removed them.
Starting with cr84 we are just copying them directly from Chromium to Brave (with our global string replace), so in theory they should come back.

This should be fixed in `1.11.x (C84)`
https://github.com/brave/brave-browser/wiki/Brave-Release-Schedule

**Comment by darkdh** [comment]
fixed by https://github.com/brave/brave-core/pull/5996

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.11.94 Chromium: 84.0.4147.85 (Build officiel) (64 bits)
-- | --
Révision | e20c13e958edc3707db3d26d513780a7f6331a4b-refs/branch-heads/4147@{#837}
Système d'exploitation | macOS Version 10.14.6 (assemblage 18G3020)
```

* Confirmed "Other Bookmarks" is translated in various places (used French):
<img width="867" alt="Capture d’écran 2020-07-13 à 3 03 39 PM" src="https://user-images.githubusercontent.com/28145373/87343070-4a06c280-c51a-11ea-95bc-9107e13cfa8c.png">
<img width="868" alt="Capture d’écran 2020-07-13 à 3 04 09 PM" src="https://user-images.githubusercontent.com/28145373/87343073-4b37ef80-c51a-11ea-8f4d-0013a9a159a5.png">
<img width="867" alt="Capture d’écran 2020-07-13 à 3 05 18 PM" src="https://user-images.githubusercontent.com/28145373/87343078-4c691c80-c51a-11ea-9459-caaed20fb9a0.png">

**Comment by Johann999** [comment]
is corrected in version 1.11.97. many thanks.

**Comment by LaurenWags** [comment]
Thank you for the update and confirmation @Johann999
